{"uuid": "293cf5ac-ea32-4f59-bc6e-73a9e52da896", "children": ["18543446-4d02-41dc-9894-52383a13293c"], "befores": [{"name": "_Class__pytest_setup_method", "status": "passed", "start": 1553479407259, "stop": 1553479416951}], "afters": [{"name": "_Class__pytest_setup_method::0", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestCustomerManageAdd' object has no attribute 'driver'\n", "trace": "  File \"D:\\softwore\\python372\\lib\\site-packages\\allure_commons\\_allure.py\", line 169, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"D:\\softwore\\python372\\lib\\site-packages\\_pytest\\fixtures.py\", line 790, in _teardown_yield_fixture\n    next(it)\n  File \"D:\\softwore\\python372\\lib\\site-packages\\_pytest\\python.py\", line 777, in xunit_setup_method_fixture\n    _call_with_optional_argument(func, method)\n  File \"D:\\softwore\\python372\\lib\\site-packages\\_pytest\\python.py\", line 691, in _call_with_optional_argument\n    func()\n  File \"D:\\my-git-project\\91lng\\selenium_91lng_auto_test\\test_case\\test_bpm\\test_customer_manage.py\", line 49, in teardown_method\n    self.driver.close()\n"}, "start": 1553479449123, "stop": 1553479449124}], "start": 1553479407259, "stop": 1553479449124}